README.md

```diff
@@ -393,26 +403,26 @@ http://kovan.dex.top/v1/depth/ETH_ADX/5
 
 ## GetActiveOrders
 
-Get user's unfilled orders or partical filled orders on current wallet address
+Get unfilled orders or partial filled orders on current wallet address.
```

Comment:
Get unfilled or partially filled orders that have not been cancelled or expired of a trader.